src/Util.js

```diff
@@ -0,0 +1,4 @@
+export default function ConvertToDate(props) {
+  const date = new Date(props.time).toLocaleDateString();
+  return date;
+}
```

Comment:
I thought it's util function to get local date string, not a  react-component.
as i know, for react component, we need return a react.node not a pure string, that's what I mention above, use lower case for `convertToDate`

Reply:
since this is used to get locale date string, you could treat this as pure js function.

Reply:
ok, I will modify it to pure JS function.